Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN dnf install -y python3 python3-requests && \
     rm -rf /var/cache/{yum,dnf}/*
 
 RUN mkdir -p /var/lib/ironic-inspector && \
-    sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal"
+    sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal" && \
+    dnf remove -y sqlite
```

Comment:
Should the dnf clean commands also be called after this operation ? to remove the cache for example, or is it not needed ?

Reply:
thanks for the suggestion
I think I will just compress the two RUN commands in a single one and put the cache clean commands at the very end